daprdocs/content/en/developing-applications/building-blocks/actors/howto-actors.md

```diff
@@ -111,6 +111,28 @@ The following request body configures a reminder with a `dueTime` 15 seconds and
 }
 ```
 
+[ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations) can also be used to specify `period`. The following request body configures a reminder with a `dueTime` 0 seconds an `period` of 15 seconds.
```

Comment:
I would add a link here to the community video on this work. 
This is the link https://www.youtube.com/watch?v=B_vkXqptpXY&t=1002s at the right time spot. 

Something like "Watch is video for more information on ISO861 for Reminders"